plugins/mumble_plugin_win32.h

```diff
@@ -18,9 +18,24 @@
 
 #include "mumble_plugin.h"
 
+#if _MSC_VER < 1800 && defined(M_IX86)
+typedef unsigned long long uintptr64_t;
+typedef unsigned long uintptr32_t;
+#else
+#include <stdint.h>
+typedef uint64_t uintptr64_t;
```

Comment:
I would prefer the types to be named in a way that makes it clear those aren't standard types or actual "internal" pointers. Maybe something like moduleptr32_t or something like that. Not that great either but you see what I mean.